validator/client/validator_attest.go

```diff
@@ -172,7 +146,14 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		}
 		return
 	}
-
+	indexedAtt.Signature = sig
+	if err := v.postSignatureUpdate(ctx, indexedAtt, pubKey); err != nil {
+		log.WithFields(logrus.Fields{
+			"sourceEpoch": indexedAtt.Data.Source.Epoch,
+			"targetEpoch": indexedAtt.Data.Target.Epoch,
+		}).Fatal("made a slashable attestation, found by external slasher service")
```

Comment:
"made a slashable attestation" could be scary for a log haha, maybe it can be changed to "prevented slashable attestation" or something? 
Also, `found by external slasher service` could be misleading if theyre not using external protection right?

Reply:
@0xKiwi thats why its fatal as it is the post attestation that have detected the slashing after it has been transmitted to the network